src/main/resources/archetype-resources/silo/src/test/resources/application-testing.yml

```diff
@@ -1,13 +1,28 @@
+stage: testing
+
 spring:
     main:
         banner-mode: "off"
 
     jpa:
         hibernate:
             ddlAuto: none
+        properties:
+            hibernate:
+                show_sql: false
+                format_sql: true
+
+    datasource:
+        url: jdbc:tc:postgresql:11://localhost/database?currentSchema=payment_reconciliation
```

Comment:
The schema should be a placeholder, right?

Reply:
öhhh :badpokerface: